extensions/wikia/PortableInfoboxBuilder/PortableInfoboxBuilderHooks.class.php

```diff
@@ -22,4 +22,111 @@ public static function onSkinAfterBottomScripts( $skin, &$text ) {
 
 		return true;
 	}
+
+	/**
+	 * Hook that exports url to the current template page
+	 *
+	 * @param Array $vars - (reference) js variables
+	 * @param Array $scripts - (reference) js scripts
+	 * @param Skin $skin - skins
+	 * @return Boolean True - to continue hooks execution
+	 */
+	public static function onWikiaSkinTopScripts( &$vars, &$scripts, $skin ) {
+		$title = $skin->getTitle();
+
+		if ( $title && $title->isSpecial( PortableInfoboxBuilderSpecialController::PAGE_NAME ) ) {
+			$templateTitleText = self::getUrlPath( $titleText = $title->getText() );
```

Comment:
I don't think we should do this.
>Space is cheap, and you're a fast typist

https://www.mediawiki.org/wiki/Manual:Coding_conventions/PHP#Assignment_expressions